Examples/Swift/Base.lproj/Main.storyboard

```diff
@@ -27,16 +27,6 @@
                                 <gestureRecognizers/>
                                 <userDefinedRuntimeAttributes>
                                     <userDefinedRuntimeAttribute type="string" keyPath="styleURL__" value="mapbox://styles/mapbox/navigation-preview-day-v2"/>
-                                    <userDefinedRuntimeAttribute type="number" keyPath="latitude">
-                                        <real key="value" value="0.0"/>
```

Comment:
Removing these user-defined runtime attributes probably resolves an issue I’d been seeing where we start at Null Island at launch and quickly ease to the user’s location.

Reply:
I have seen so many issues when this is added....